### DISCUSSION TITLE
How to handle the aspect ratio on a resizable form

### DISCUSSION THREAD

**Discussion by team-moeller** [team-moeller/better-access-charts#83] [Ideas]
Original question from @KDGundermann

we have to discuss how we should handle the problem that Access does not update the height / width attribute of the browser control.

Option 1 : give a code example only in the docs how to handle the problem
Option 2 : include code in BAC to show how to handle the problem

**Comment by KDGundermann** [comment]
My solution need 3 additional classes:
- FI_Form, FI_Controls, FI_Control

to track the Form_Resize Event
( They are coming from my library where I track Form_Resize to resize the controls too )
